test/transformer-compile-class.test.ts

```diff
@@ -177,4 +177,23 @@ describe('transformer-compile-class', () => {
 
     expect(invalidateFn).toHaveBeenCalledTimes(2)
   })
+
+  it('test border', async () => {
+    const result = await transform(`
+    <div class="border-[calc(var(--border-width)*1px)]"/>
```

Comment:
The location file of the test case is incorrect, it should be in `test/assets/preset-mini-targets.ts`

Reply:
thx~